Source/Resources/Compute/VirtualMachine/VirtualMachine.ps1

```diff
@@ -0,0 +1,114 @@
+. $PSScriptRoot/../../Common/Common.ps1
+
+function Get-VirtualMachine {
+    param(
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNull()]
+        [PSObject] $Definition,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [String] $Name
+    )
+
+    $contextObject = $Definition.contexts.default
```

Comment:
Are you planning to make this possible to override? How do I do if I want to check something on a VM that is not in default context?

Reply:
Context added